src/pages/workspace/WorkspaceMembersPage.js

```diff
@@ -371,16 +371,19 @@ class WorkspaceMembersPage extends React.Component {
     }
 
     render() {
-        const policyMemberList = lodashGet(this.props, 'policyMemberList', {});
         const policyOwner = lodashGet(this.props.policy, 'owner');
         const currentUserLogin = lodashGet(this.props.currentUserPersonalDetails, 'login');
         const removableMembers = {};
         let data = [];
-        _.each(policyMemberList, (policyMember, email) => {
+        _.each(this.props.policyMembers, (policyMember, accountID) => {
             if (this.isDeletedPolicyMember(policyMember)) {
                 return;
             }
-            const details = lodashGet(this.props.personalDetails, email, {displayName: email, login: email});
+            const details = this.props.personalDetails[accountID];
+            if (!details) {
+                Log.hmmm(`[WorkspaceMembersPage] no personal details found for policy member with accountID: ${accountID}`);
+                return;
+            }
```

Comment:
This caused a regression https://github.com/Expensify/App/issues/22442. When we add a new user offline we didn't set their optimistic data in `personalDetails` but we used the fallback object (the one in `lodashGet`). Now neither is available which made the offline behaviour broken (user not added to the members list).